Shared/BasemapPickerController.h

```diff
@@ -70,12 +74,15 @@ public slots:
   void basemapsDataPathChanged();
   void basemapChanged(Esri::ArcGISRuntime::Basemap* basemap, QString name = "");
   void toolErrorOccurred(const QString& errorMessage, const QString& additionalMessage);
+  void selectedBasemapIndexChanged();
 
 private:
   TileCacheListModel* m_tileCacheModel;
   int                 m_defaultBasemapIndex = 0;
   QString             m_basemapDataPath;
   QString             m_defaultBasemap = "topographic";
+  int                 m_selectedBasemapIndex = -1;
+  QString             m_selectedBasemapPath = "";
```

Comment:
```suggestion
  QString             m_selectedBasemapPath;
```
Default constructor does the same thing, perhaps more efficiently.